bundler/spec/commands/newgem_spec.rb

```diff
@@ -698,6 +698,33 @@ def create_temporary_dir(dir)
         to match(/example\.com/)
     end
 
+    context "git config github.user present" do
+      before do
+        bundle "gem #{gem_name}"
+      end
+
+      it "gemspec changelog_uri set to include git username" do
+        expect(generated_gemspec.metadata["changelog_uri"]).
+          to eq("https://github.com/bundleuser/#{gem_name}/blob/main/CHANGELOG.md")
+        expect(generated_gemspec.metadata["homepage_uri"]).to eq("https://github.com/bundleuser/#{gem_name}")
```

Comment:
we should probably test for both generated_gemspec.metadata["homepage_uri"] and also `generated_gemspec.homepage` in both branches since they're the same in the generated gemspec.